### DISCUSSION TITLE
blammed does something annoying

### DISCUSSION THREAD

**Discussion by DeceptedPar** [DEMOLITIONDON96/Demolition-Engine#114] [General (Friday Night Chattin')]
I was going to play blammed and then before the song started in the intro cmd pops up and bring me to cornhub.com
why... just why...
please remove this it's annoying

**Comment by DeceptedPar** [comment]
you could just get rid of test.lua in the blammed chart folder

**Reply by RealDealJinx** [reply]
people will suffer.
jk i forgot to remove the folder
it was a test to see if browser loads work
